<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeAuto</class>
 <widget class="QWidget" name="ChangeAuto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="return_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>return</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="templcd">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="ohmSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="tempBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
width: 20px;
height: 20px;
}</string>
   </property>
   <property name="text">
    <string>enable</string>
   </property>
  </widget>
  <widget class="QLabel" name="templabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Temperatur in Â°C</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>70</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ground humidity in %</string>
   </property>
  </widget>
  <widget class="QSlider" name="mintempSlider">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>-10</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
   <property name="sliderPosition">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="ohmBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
width: 20px;
height: 20px;
}</string>
   </property>
   <property name="text">
    <string>enable</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLCDNumber" name="ohmlcd">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>150</width>
     <height>70</height>
    </rect>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="maxtempSlider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="minimum">
    <number>-10</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>30</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>max</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
